components/formats-bsd/test/loci/formats/utests/MapAnnotationTest.java

```diff
@@ -129,14 +130,21 @@ public void testMapAnnotationValueContent() throws Exception {
     assertNotNull(ome.getImage(0).getLinkedAnnotation(0)); 
 
     MapAnnotation mapAnnotation = (MapAnnotation) ome.getImage(0).getLinkedAnnotation(0); 
-    MapPairs dataMap = mapAnnotation.getValue();
-
-    assertEquals(5, dataMap.getPairs().size());
-    assertEquals(new MapPair("a", "1"), dataMap.getPairs().get(0));
-    assertEquals(new MapPair("b", "2"), dataMap.getPairs().get(1));
-    assertEquals(new MapPair("c", "3"), dataMap.getPairs().get(2));
-    assertEquals(new MapPair("d", "4"), dataMap.getPairs().get(3));
-    assertEquals(new MapPair("e", "5"), dataMap.getPairs().get(4));
+    LinkedListMultimap<String,String> dataMap = mapAnnotation.getValue();
+
+    assertEquals(6, dataMap.size());
+    assertEquals("a", dataMap.entries().get(0).getKey());
+    assertEquals("1", dataMap.entries().get(0).getValue());
+    assertEquals("d", dataMap.entries().get(0).getKey());
+    assertEquals("2", dataMap.entries().get(0).getValue());
+    assertEquals("c", dataMap.entries().get(0).getKey());
+    assertEquals("3", dataMap.entries().get(0).getValue());
+    assertEquals("b", dataMap.entries().get(0).getKey());
+    assertEquals("4", dataMap.entries().get(0).getValue());
+    assertEquals("e", dataMap.entries().get(0).getKey());
+    assertEquals("5", dataMap.entries().get(0).getValue());
+    assertEquals("c", dataMap.entries().get(0).getKey());
+    assertEquals("6", dataMap.entries().get(0).getValue());
```

Comment:
Ah, I was using netbeans, but it may actually be a problem with my project setup.